library/src/main/java/com/doomonafireball/betterpickers/radialtimepicker/AmPmCirclesView.java

```diff
@@ -38,8 +39,7 @@
     private static final String TAG = "AmPmCirclesView";
 
     // Alpha level for selected circle.
-    private static final int SELECTED_ALPHA = Utils.SELECTED_ALPHA;
-    private static final int SELECTED_ALPHA_THEME_DARK = Utils.SELECTED_ALPHA_THEME_DARK;
+    private static final int SELECTED_ALPHA = 51;
```

Comment:
I'll clean this up. I left some values behind that I shouldn't have. SELECTED_ALPHA comes from the style now.